chainer/training/extensions/warmup_shift.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import division
+
+from chainer.training import extension
+
+
+class WarmupShift(extension.Extension):
+
+    """Trainer extension to change an optimizer attribute evenly at the
+    begining of one training.
+
+    For example, suppose that this extension is called at every iteration,
+    and warmup_start_lr = x , init = y, warmup_iter = t.
```

Comment:
How about naming it `warmup_start` instead of `warmup_start_lr`, as (strictly speaking) it can be used for values other than learning rates?